.github/dependabot.yml

```diff
@@ -14,3 +14,20 @@ updates:
       security-updates:
         applies-to: security-updates
         dependency-type: production
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    assignees:
+      - wendigo
+    open-pull-requests-limit: 10
```

Comment:
Why do we need 10 open PRs about dependency upgrade ?
Can't we have it limited to 2 or 3 ?

Reply:
Let's see in a week if the amount of PRs is too high